app/src/androidTest/java/com/github/swent/echo/compose/authentication/createProfileTest.kt

```diff
@@ -0,0 +1,72 @@
+package com.github.swent.echo.compose.authentication
+
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onAllNodesWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.navigation.compose.rememberNavController
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.swent.echo.data.model.SectionEPFL
+import com.github.swent.echo.data.model.SemesterEPFL
+import com.github.swent.echo.data.model.Tag
+import com.github.swent.echo.ui.navigation.NavigationActions
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class CreateProfileTest {
+    @get:Rule val composeTestRule = createComposeRule()
+
+    @Test
```

Comment:
Before adding your @Test, you should define your composeTestRule.setContent inside a @Before fun setup() function in order to instantiate your composable before each @Test. Take a look at the test done by the other to have examples of the correct structure.

Reply:
I didn't put it in a @before because there was only 1 test case XD